### PR TITLE
drop net-http01 prow jobs

### PR DESCRIPTION
**Pull request by dprotaso**
/assign @ReToCode @skonto 

Part of https://github.com/knative/serving/issues/14640

### CODE REVIEW

File: config/prow/k8s-testgrid/k8s-testgrid.yaml [hunk 1]
```diff
@@ -88,7 +87,6 @@ dashboards:
 - name: knative-extensions-release-1.0
 - name: knative-extensions-release-1.1
 - name: knative-extensions-release-1.10
-- name: knative-extensions-release-1.11
```
Comment (izabelacg) [inline config/prow/k8s-testgrid/k8s-testgrid.yaml:91]: is this line removal intentional?

Comment (dprotaso) [reply config/prow/k8s-testgrid/k8s-testgrid.yaml:91]: It's autogenerated. I believe it is - 1.11 was the last net-http01 release so dropping the jobs as well.

Comment (izabelacg) [reply config/prow/k8s-testgrid/k8s-testgrid.yaml:91]: oh, got it. Thanks

**Pull request merged**